util/pkg/vfs/s3fs.go

```diff
@@ -188,6 +196,53 @@ func (p *S3Path) ReadFile() ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+// GetVersioning implements Path::GetVersioning
```

Comment:
> GetVersioning implements Path::GetVersioning

I am thinking that you did not implement this call in the interface.  Please update the comment.